extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -37,7 +38,8 @@ define('ext.wikia.adEngine.lookup.services', [
 			}
 		},
 		bidMarker = ['x', 'x', 'x', 'x', 'x'],
-		realSlotPrices = {};
+		realSlotPrices = {},
+		scrollY = {};
```

Comment:
Can we find better module for this information? Like `slotRegistry` which is already holding the refresh count (`rv` parameter)